client/src/components/ReUsableComponents/RequiredButtons.tsx

```diff
@@ -0,0 +1,13 @@
+import Button, { ButtonProps } from '@mui/material/Button';
+import { purple } from '@mui/material/colors';
+import { styled } from '@mui/material/styles';
+
+const RequiredButton = styled(Button)<ButtonProps>(({ theme }) => ({
```

Comment:
The name `RequiredButton` is interesting. Could you elaborate on the naming for this?